releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1365,10 +1514,30 @@
     "do_not_publish": true,
     "is_mapped": true
   },
+  "124061": {
+    "commit": "04d2f336419b5a824cb96cb88462ef18a90d619d",
+    "text": "CustomResourceDefinition objects created with non-empty `caBundle` fields which are invalid or do not contain any certificates will not appear in discovery or serve endpoints until a valid `caBundle` is provided. Updates to CustomResourceDefinition are no longer allowed to transition a valid `caBundle` field to an invalid `caBundle` field, because this breaks serving of the existing CustomResourceDefinition.",
+    "markdown": "CustomResourceDefinition objects created with non-empty `caBundle` fields which are invalid or do not contain any certificates will not appear in discovery or serve endpoints until a valid `caBundle` is provided. Updates to CustomResourceDefinition are no longer allowed to transition a valid `caBundle` field to an invalid `caBundle` field, because this breaks serving of the existing CustomResourceDefinition. ([#124061](https://github.com/kubernetes/kubernetes/pull/124061), [@Jefftree](https://github.com/Jefftree)) [SIG API Machinery]",
+    "author": "Jefftree",
+    "author_url": "https://github.com/Jefftree",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/124061",
+    "pr_number": 124061,
+    "areas": [
+      "apiserver"
+    ],
+    "kinds": [
+      "bug",
+      "api-change"
+    ],
+    "sigs": [
+      "api-machinery"
+    ],
+    "duplicate_kind": true
+  },
   "124063": {
     "commit": "a7ece470e56f0479573db2632031d274867d96b1",
-    "text": "Fixes a bug where hard evictions due to resource pressure would let the pod have the full termination grace period, instead of shutting down instantly. This bug also affected force deleted pods. Both cases now get a termination grace period of 1 second.",
-    "markdown": "Fixes a bug where hard evictions due to resource pressure would let the pod have the full termination grace period, instead of shutting down instantly. This bug also affected force deleted pods. Both cases now get a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr)) [SIG Node]",
+    "text": "Fixed a bug where hard evictions due to resource pressure allowed pods\nto use the full termination grace period instead of shutting down instantly. \nThis bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second.",
+    "markdown": "Fixed a bug where hard evictions due to resource pressure allowed pods\n  to use the full termination grace period instead of shutting down instantly. \n  This bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr))",
```

Comment:
```suggestion
    "markdown": "Fixed a bug where hard evictions due to resource pressure allowed pods\n  to use the full termination grace period instead of shutting down instantly. \n  This bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr)) [SIG Node]",
```